decidim-participatory_processes/app/helpers/decidim/participatory_processes/participatory_process_helper.rb

```diff
@@ -76,6 +77,32 @@ def process_nav_items(participatory_space)
           }
         end
       end
+
+      def filter_sections
+        [
+          { method: :with_date, collection: filter_dates_values, label_scope: "decidim.participatory_processes.participatory_processes.filters", id: "date" },
+          { method: :with_scope, collection: filter_global_scopes_values, label_scope: "decidim.meetings.meetings.filters", id: "scope" },
+          { method: :with_area, collection: filter_areas_values, label_scope: "decidim.initiatives.initiatives.filters", id: "area" },
+          { method: :with_type, collection: filter_types_values, label_scope: "decidim.participatory_processes.participatory_processes.filters", id: "type" }
+        ].reject { |item| item[:collection].blank? }
+      end
+
+      def process_types
+        @process_types ||= Decidim::ParticipatoryProcessType.joins(:processes).distinct
+      end
+
+      def filter_types_values
+        return if process_types.blank?
+
+        type_values = process_types.map { |type| [type.id.to_s, filter_text_for(translated_attribute(type.title))] }
+        type_values.prepend(["", filter_text_for(t("decidim.meetings.meetings.filters.type_values.all"))])
```

Comment:
Let's avoid using meetings translations keys in ParticipatoryProcesses.

Reply:
Fixed